samples/XCT.Sample/Pages/Views/Calendar/DateTimeOffsetToDayConverter.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Sample.Pages.Views
+{
+	public class DateTimeOffsetToDayConverter : IValueConverter
+	{
+		public object Convert(object? value, Type? targetType, object? parameter, CultureInfo? culture)
+		{
+			if (value == null)
```

Comment:
The `null` check will not prevent the possible `InvalidCastException` at line 14
```suggestion
			if (value is not DateTimeOffset)
```